src/style.css

```diff
@@ -0,0 +1,122 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  width: 100%;
+  display: flex;
+  /* border: 5px solid orange; */
+}
+
+.block {
+  min-width: 300px;
+  height: 300px;
+  flex-grow: 0;
```

Comment:
this css property is useful in your case(it's overwritten with the media query styles)